dapps/assets/src/components/SearchService.ts

```diff
@@ -17,4 +17,86 @@
   the following URL: https://evan.network/license/
 */
 
-export class SearchService {}
+import axios from 'axios';
+import { utils } from '@evan.network/api-blockchain-core';
+
+let agentUrl = 'https://search.test.evan.network/api/smart-agents' // TODO: prod/dev switch
```

Comment:
```suggestion
let agentUrl;
```

define the searchUrl when the runtime is available